infra/experimental/sanitizers/ExecSan/execSan.cpp

```diff
@@ -60,6 +62,86 @@
 const std::string kTripWire = "/tmp/tripwire";
 // The PID of the root process we're fuzzing.
 pid_t g_root_pid;
+// Assuming the longest pathname is "/bin/bash".
+const int kShellPathnameLength = 10;
```

Comment:
nit: use constexpr for these: https://google.github.io/styleguide/cppguide.html#Use_of_constexpr